test/libsolidity/syntaxTests/constants/define_constants_with_external_constant_member_var.sol

```diff
@@ -0,0 +1,11 @@
+uint256 constant MAX = 1;
+
+library Internal {
+    uint256 internal constant SCALE = 100;
+}
+
+contract Usage {
```

Comment:
Let's switch to our typical naming that lets you see at a glance what is a contract, what is a library, etc.

```suggestion
library L {
    uint256 internal constant SCALE = 100;
}

contract C {
```

Reply:
Done